### ISSUE TITLE
Time doesn't change

### ISSUE THREAD

**Issue by dfsmirnov** [xdan/periodpicker#71]
Same config as in #70 
In opened picker click on a date. Than click on a hour (without mousewheel). Than click on a minute.
After 'ok' we have changed date but, not changed, old time

**Comment by dfsmirnov** [comment]
There is small video https://youtu.be/ajPpERy4aXk with this bug

Sometimes it works, sometimes it does not work

**Comment by kclark6595** [comment]
I am having the same issue.  Time works fine only if input is initially blank and time is picker before date.  My config is:

```
            $('#timepicker').periodpicker({
                likeXDSoftDateTimePicker: true,
                hideOnBlur: false,
                hideAfterSelect: false,
                norange: true, // use only one value
                cells: [1, 1], // show only one month
                dayOfWeekStart: 7,

                resizeButton: false, // deny resize picker
                fullsizeButton: false,
                fullsizeOnDblClick: false,

                showDatepickerInputs: false,
                showTimepickerInputs: false,
                timePickerInRight: true,
                formatDateTime: 'M/D/YYYY h:mm a',

                timepicker: true, // use timepicker
                timepickerOptions: {
                    saveOnChange: true,
                    steps:[1,15,1,1],
                    hours: true,
                    minutes: true,
                    seconds: false,
                    ampm: true
                }
            });
```
Anyone found a fix yet?

**Comment by xdan** [comment]
Hi, fixed in 6.2.6 Please download latest version https://xdsoft.net/jqplugins/periodpicker/?update#plugin_sender
